CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Next release
 
+### Added
+
+- Insights edit view export functionality
```

Comment:
When adding items to the changelog, we should have GS and biz dev in mind. Will they understand this? Ideally, you should describe where they can find this, and a little bit more description of what this feature does.

Reply:
Good point, I'll add a bit more details to this one, thanks!